ansible/local_scripts/goinstall.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -e
+
+VERSION="1.8.1"
+
+print_help() {
+    echo "Usage: bash goinstall.sh OPTION"
+    echo -e "\nOPTIONS:"
+    echo -e "  --32\t\tInstall 32-bit version"
```

Comment:
@Dual2Core - are all these options required? If not - [delete them](https://martinfowler.com/bliki/Yagni.html), please.